src/aics-image-viewer/components/TfEditor/index.tsx

```diff
@@ -72,14 +79,36 @@ const TF_GENERATORS: Record<string, (histogram: Histogram) => Lut> = {
   resetXF: (_histo) => new Lut().createFullRange(),
 };
 
+// *---*
+// |   |
+// |   |
+//  \ /
+//   *
+// width: 0.65 * height; height of rectangle: 0.6 * height; height of triangle: 0.4 * height
```

Comment:
❤️  love this -- but semi-wondering if a svg does any better here.

Reply:
This is an svg! The context works like a canvas, but D3 turns it into an svg data string.